internal/core/api.rs

```diff
@@ -264,25 +406,22 @@ pub use crate::input::PointerEventButton;
 #[non_exhaustive]
 pub enum WindowEvent {
     /// The mouse or finger was pressed
-    PointerPressed { position: euclid::Point2D<f32, LogicalPx>, button: PointerEventButton },
+    PointerPressed { position: LogicalPosition, button: PointerEventButton },
     /// The mouse or finger was released
-    PointerReleased { position: euclid::Point2D<f32, LogicalPx>, button: PointerEventButton },
+    PointerReleased { position: LogicalPosition, button: PointerEventButton },
     /// The position of the pointer has changed
-    PointerMoved { position: euclid::Point2D<f32, LogicalPx> },
+    PointerMoved { position: LogicalPosition },
     /// Wheel was rotated.
     /// `pos` is the position of the mouse when the event happens.
     /// `delta` is the amount of pixel to scroll.
-    PointerScrolled {
-        position: euclid::Point2D<f32, LogicalPx>,
-        delta: euclid::Vector2D<f32, LogicalPx>,
-    },
+    PointerScrolled { position: LogicalPosition, delta: LogicalPosition },
```

Comment:
I am a bit unsure about the use of a position type for the delta.
Maybe we could split into two f32 (horizontal and vertical delta) ?

Reply:
That's fine with me as well. I'm not particularly attached to using a struct here. I like separate fields.

Reply:
Done